CHANGELOG.md

```diff
@@ -2,6 +2,41 @@
 
 All notable changes to `laravel-aws-pubsub` will be documented in this file.
 
+## 0.3.0 - 2021-11-09
+
+Adds support for publishing pub/sub events to AWS EventBridge instead of AWS SNS [#11](https://github.com/Pod-Point/laravel-aws-pubsub/pull/11)
```

Comment:
`..in addition to AWS SNS..` ?

Reply:
Do we support publishing to both at once?